container/docker/templates/conductor-src-dockerfile.j2

```diff
@@ -24,7 +24,8 @@ RUN yum -y update-minimal --disablerepo "*" \
                    https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
     yum -y install --disablerepo "*" \
                    --enablerepo epel,rhel-7-server-rpms,rhel-7-server-optional-rpms \
-                   make gcc git python-devel curl rsync libffi-devel openssl-devel tar redhat-rpm-config&& \
+                   make gcc git python-devel curl rsync libffi-devel openssl-devel tar redhat-rpm-config && \
+    yum -y remove python-chardet && \
```

Comment:
What's with this removal?

Reply:
No idea why, but the RPM check within the scanner flags the existence of the`chardet` module files within `/usr/lib/python2.7/site-pacakges/`  and fails. Removing it mollifies the scanner, and doesn't seem to break our tests.